wsclient/src/test/resources/SimpleFunction.ecl

```diff
@@ -0,0 +1,19 @@
+MyFunc(STRING DataIn, STRING1 SearchChar) := FUNCTION
```

Comment:
if the content of the query matters, we can figure out how to publish RoxieEchoPersonInfo instead.